apps/eew/sceewlog/config/sceewlog.cfg

```diff
@@ -195,4 +202,4 @@ FCM.activate = false
 # the topic name for Firebase
 # see more about Autho. Key at:https://firebase.google.com/docs/cloud-messaging/auth-server 
 # for topics: https://firebase.google.com/docs/cloud-messaging/android/topic-messaging
-FCM.dataFile = /home/sysop/seiscomp/share/sceewlog/.fcmdatafile
\ No newline at end of file
+FCM.dataFile = /home/sysop/seiscomp/share/sceewlog/.fcmdatafile
```

Comment:
@USERCONFIGDIR@/.fcmdatafile would be better but maybe in next version?

Reply:
Fred, okay but @USERCONFIGDIR@ and the other simplified way for a path are not working. That is the reason in setting a whole example path. Should it be necessary to import a SC python module?

Reply:
There must be a way, but we'll see that later